src/modules/Dropdown/Dropdown.js

```diff
@@ -425,64 +422,57 @@ export default class Dropdown extends Component {
         if (openOnFocus) this.open()
       }
       if (!this.state.open) {
-        document.addEventListener('keydown', this.openOnArrow)
-        document.addEventListener('keydown', this.openOnSpace)
-      } else {
-        document.addEventListener('keydown', this.moveSelectionOnKeyDown)
-        document.addEventListener('keydown', this.selectItemOnEnter)
+        eventPool.sub('keydown', [this.openOnArrow, this.openOnSpace])
       }
-      document.addEventListener('keydown', this.removeItemOnBackspace)
```

Comment:
`removeItemOnBackspace` works when a `multiple search selection` Dropdown is focused and `openOnFocus={false}`.  The cursor focusses the Dropdown, it does not open, but the backspace listener is added so the user can remove labels:

![http://g.recordit.co/XcbauX3d4V.gif](http://g.recordit.co/XcbauX3d4V.gif)

I'd prefer we don't change any logic in this PR except the adding/removing events.  That said, it looks like we're missing a test for this as well.  Changing that logic should have broken a test.